packages/styled-components/src/components/primitives/Chat/Conversation/Message/Message.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Message } from '.';
+import { action } from '@storybook/addon-actions';
+import { text, select } from '@storybook/addon-knobs';
+
+export default {
+  title: 'Primitives/Chat/Conversation/Message',
+  component: Message,
+};
+
+const mediaFile = {
+      contentType: 'image/jpeg',
+      url:
+        'https://www.autobild.es/sites/autobild.es/public/styles/main_element/public/dc/fotos/tesla_model_S_001.jpg?itok=jttOEAGK',
```

Comment:
please use the composer logo as in the other stories

Reply:
done!.